app/forms/gobierto_admin/gobierto_people/settings_form.rb

```diff
@@ -65,6 +75,14 @@ def save_settings
           settings_attributes.home_text_en = home_text_en
           settings_attributes.submodules_enabled = submodules_enabled.select{|m| m.present?}
           settings_attributes.calendar_integration = calendar_integration
+
```

Comment:
I think we should add a validation in the form: if the user selects IBM Notes integration but user and password are empty, we should display an error message that credentials are required.